docs/evolutionaryAlgorithm/README.md

```diff
@@ -0,0 +1,264 @@
+
+But first, prepare the data source directory in the runtime's `imports` folder. For example, for data source named `villeray`, make sure you have the following file hierarchy:
+
+```
+<Transition runtime dir>/
+ |-- imports/
+    |-- villeray/
+       |-- polygon.geojson
+```
+
+First, a `polygon.geojson` file containing the polygon for which to import the data needs to be available in the data source directory. [geojson.io](https://geojson.io) can be used to create one.
+
+
+The following steps need to be done to prepare the data: 
+
+*Note*: all those steps can be done non-interactively, ie by passing each parameter on the command line. But in the absence of a proper way for tasks to announce their parameters and the lack of documentation for it, the current approach is to read to code and find out (and update this documentation!) or run it interactively. A frequent parameter is to use `--dataSourceId <name of data source>` that works on most tasks.
+
+1. Download the data from open street map:
+
+```
+$ yarn node packages/chaire-lib-backend/lib/scripts/osm/1_downloadOsmDataToFiles.task.js
+yarn node v1.22.22
+Using .env file from ./.env
+Read server configuration from ./examples/config.js
+? Please select a data source for import (Use arrow keys)
+❯ villeray 
+  --New data source--
+? Please select a data source for import 
+Retrieving data from osm...
+Done retrieving data from osm...
+Writing osm data to runtime/imports/villeray/osmRawData.json
+Done writing osm data
+Retrieving data from osm...
+Done retrieving data from osm...
+Writing osm geojson data to runtime/imports/villeray/osmData.geojson
+Done writing osm geojson data
+```
+
+2. (optional) Enhance the polygon data. it adds the polygon area in sq meters, for polygons, it calculates the area and the floor area using the building:levels value and the calculated area, or it takes the existing floor area from the building:floor_area tag, which is supposed to be in sq meters.
+```
+yarn node packages/chaire-lib-backend/lib/scripts/osm/1b_prepareAndEnhanceOsmPolygonData.task.js
+? Please select a data source for import (Use arrow keys)
+❯ villeray 
+  --New data source--
+Running import in dir runtime/imports/villeray/
+Enhancing and saving polygon geojson data to file:  runtime/imports/villeray/osmPolygonsEnhanced.geojson
+Fetch buildings from osm geojson data...
+There are 16103 buildings in the osm geojson data.
+Fetch polygons and multipolygons from osm geojson data...
+=== validated and enhanced polygon 16970 ===
+Finished with 16971 polygons.
+```
+
+3. Import and process osm data to files. This will create 3 geojson files, one for points of interest, one for residential buildings and one for residential zones. This task will give a number of warnings on buildings and places that may have missing or incomplete data and for which an approximation is used. It is worth running this task in non-interactive mode and pipe the output to a file for later processing
```

Comment:
The only thing I'm missing is why do I want to do some later processing with the redirected output? I don't see use case for that described later in the file. Maybe mention an example here??

Reply:
J'ai passé la fin de semaine à mettre des points dans les bons buildings et ajouter des entrées à des magasins! C'est ça que ça veut dire!! :D

Reply:
Ah ok!  Tu peux ajouter (later processing == fixing/editing the OSM data)